ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -816,13 +816,19 @@ def _get_connections_core(
             instrument_type_id, self._reserved_instrument_type_ids
         )
 
+        requested_pin_or_relay_names = _to_ordered_set(
```

Comment:
```suggestion
        resolved_pin_or_relay_names = _to_ordered_set(
```
Please share your thoughts

Reply:
I have shared opinions on this. In one way, we could take it like they're still pins or relays (at the lowest level) that the user has reserved. Thinking in this particular context, it is the requested pin or relay names that is resolved now. 

Maybe `requested_pin_or_relay_names` sounds good in that way.

Reply:
Updated.